test/unit/spec/v2/client/startLoginFlow_spec.js

```diff
@@ -0,0 +1,161 @@
+import Settings from 'models/Settings';
+import { startLoginFlow } from 'v2/client/startLoginFlow';
+import sessionStorageHelper from 'v2/client/sessionStorageHelper';
+
+jest.mock('v2/client/interact', () => {
+  return {
+    interact: () => { }
+  };
+});
```

Comment:
style-nit: Future versions of the eslint-jest plugin warn when we have return objects like this. Instead, consider the shorthand:

```suggestion
jest.mock('v2/client/interact', () => ({
  interact: () => { }
}));
```

Reply:
I'd defer to eslint to auto-fix when we have chance to tweak eslint rule.